sdk/python/feast/feature_server.py

```diff
@@ -61,11 +66,13 @@ async def get_online_features(request: Request):
     return app
 
 
-def start_server(store: "feast.FeatureStore", host: str, port: int):
+def start_server(
+    store: "feast.FeatureStore", host: str, port: int, no_access_log: bool
+):
     app = get_app(store)
     click.echo(
         "This is an "
         + click.style("experimental", fg="yellow", bold=True, underline=True)
         + " feature. It's intended for early testing and feedback, and could change without warnings in future releases."
     )
-    uvicorn.run(app, host=host, port=port)
+    uvicorn.run(app, host=host, port=port, access_log=(not no_access_log))
```

Comment:
Hey @nossrannug , thanks for your PR. But I'm confused, what's the benefit of moving to the synchronous interface if we'll continue to use uvicorn as our server and at the end of the day we'll have a single-threaded app.

Reply:
Did you use some other server in your tests?

Reply:
Hey @pyalex,
We're not running a single-threaded app. FastAPI supports mixing async and sync methods(endpoints) as much as you want and it will take care of running it correctly. When a method is async it will be executed using the event loop, but when a method is synchronous it will be executed in a thread pool.

sync:
![image](https://user-images.githubusercontent.com/416906/145673369-62f3f677-684e-4b12-82fd-03873ddbf8f8.png)

async:
![image](https://user-images.githubusercontent.com/416906/145673397-aaf20a9c-6732-44e5-ad7d-f31e76af31f4.png)